src/ir/comp.rs

```diff
@@ -490,49 +490,67 @@ fn bitfields_to_allocation_units<E, I>(ctx: &BindgenContext,
     let mut unit_align = 0;
     let mut bitfields_in_unit = vec![];
 
+    // TODO(emilio): Determine this from attributes or pragma ms_struct
+    // directives. Also, perhaps we should check if the target is MSVC?
```

Comment:
In fact, we could do this right now, right? Since the mangling PR, we have access to the target triple, so we should probably jsut check if "msvc" is in it or something like that here.